content/tasks.py

```diff
@@ -1,7 +1,43 @@
+from datetime import datetime
+import operator
+
 from djcelery import celery
 from content.forms import render_mathml
+from content.models import Event, EventParticipantRel, EventQuestionRel
+from core.models import Participant
+from organisation.models import CourseModuleRel
 
 
 @celery.task
 def render_mathml_content():
-    render_mathml()
\ No newline at end of file
+    render_mathml()
+
+
+@celery.task
+def end_event_processing():
+    events = Event.objects.filter(deactivation_date__gt=datetime.now(), end_processed=False)
+    for event in events:
+        participants = EventParticipantRel.objects.filter(event=event)
+
+        scores = {}
+        for participant in participants:
+            scores[participant.id] = EventQuestionRel.objects.filter(event=event, participant=participant, correct=True) \
+                .count()
```

Comment:
20-25: Should be done with a single annotate
EventQuestionRel.objects.filter(...).annotate(score=Count("pk"))